src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx

```diff
@@ -123,11 +94,12 @@ export const OnboardingDetailPage: React.FC = () => {
         >
           <Tab
             id={TabId.CREATE_A_FLAG}
-            panel={
-              <CreateAFlagView flagInfo={flagInfo} setFlagName={setFlagName} isCreated={flagCreated} goNext={onNext} />
-            }
+            panel={<CreateAFlagView selectedFlag={selectedFlag} setSelectedFlag={setSelectedFlag} />}
             title={
-              <Text icon={flagCreated ? 'tick-circle' : undefined} iconProps={{ color: Color.GREEN_500, size: 14 }}>
+              <Text
+                icon={selectedFlag?.identifier ? 'tick-circle' : undefined}
```

Comment:
nitpick
could this be shortened to 
```suggestion
                icon={selectedFlag?.identifier && 'tick-circle'}
```

Reply:
Can't change this as it gives a TS error - the prop `icon` needs to be of type `IconName` or be completely omitted
<img width="787" alt="image" src="https://user-images.githubusercontent.com/5584348/190458779-7f0605bf-3cde-42bb-a5d2-159465ab9359.png">